cmslitemetadata_to_redshift/cmslitemetadata_to_redshift.py

```diff
@@ -575,434 +576,8 @@ def report(data):
-        ELSE NULL
-      END AS level3_id,
-      CASE
-        WHEN
-TRIM(SPLIT_PART(full_tree_nodes, '|', 7)) =
-'FD6DB5BA2A5248038EEF54D9F9F37C4D'
-          AND TRIM(SPLIT_PART(full_tree_nodes, '|', 8)) <> ''
-          THEN TRIM(SPLIT_PART(full_tree_nodes, '|', 8))
-        WHEN
-TRIM(SPLIT_PART(full_tree_nodes, '|', 7)) <>
-'FD6DB5BA2A5248038EEF54D9F9F37C4D'
-          AND TRIM(SPLIT_PART(full_tree_nodes, '|', 5)) <> ''
-          THEN TRIM(SPLIT_PART(full_tree_nodes, '|', 5))
-        ELSE NULL
-      END AS level4_id,
-      CASE
-        WHEN
-TRIM(SPLIT_PART(full_tree_nodes, '|', 7)) =
-'FD6DB5BA2A5248038EEF54D9F9F37C4D'
-          AND TRIM(SPLIT_PART(full_tree_nodes, '|', 9)) <> ''
-          THEN TRIM(SPLIT_PART(full_tree_nodes, '|', 9))
-        WHEN
-TRIM(SPLIT_PART(full_tree_nodes, '|', 7)) <>
-'FD6DB5BA2A5248038EEF54D9F9F37C4D'
-          AND TRIM(SPLIT_PART(full_tree_nodes, '|', 6)) <> ''
-          THEN TRIM(SPLIT_PART(full_tree_nodes, '|', 6))
-        ELSE NULL
-      END AS level5_id
-    FROM {dbschema}.metadata AS cm
-      LEFT JOIN {dbschema}.metadata AS cm_parent
-        ON cm_parent.node_id = cm.parent_node_id
-    WHERE cm.page_type like 'ASSET_FOLDER'
-    OR cm.page_type LIKE 'ASSET'),
-biglist
-  AS (SELECT
-    ROW_NUMBER () OVER ( PARTITION BY ids.node_id ) AS index,
-    ids.*,
-    l1.title AS asset_theme,
-    l2.title AS asset_subtheme,
-    l3.title AS asset_topic,
-    l4.title AS asset_subtopic,
-    l5.title AS asset_subsubtopic,
-  CASE
-    WHEN asset_theme IS NOT NULL
-      THEN level1_ID
-    ELSE NULL
-  END AS asset_theme_ID,
-  CASE
-    WHEN asset_subtheme IS NOT NULL
-      THEN level2_ID
-    ELSE NULL
-  END AS asset_subtheme_ID,
-  CASE
-    WHEN asset_topic IS NOT NULL
-      THEN level3_ID
-    ELSE NULL
-  END AS asset_topic_ID,
-  CASE
-    WHEN asset_subtopic IS NOT NULL
-      THEN level4_ID
-    ELSE NULL
-  END AS asset_subtopic_ID,
-  CASE
-    WHEN asset_subsubtopic IS NOT NULL
-      THEN level5_ID
-    ELSE NULL
-  END AS asset_subsubtopic_ID,
-  l1.sitekey as sitekey
-FROM ids
-    LEFT JOIN {dbschema}.metadata AS l1
-      ON l1.node_id = ids.level1_id
-    LEFT JOIN {dbschema}.metadata AS l2
-      ON l2.node_id = ids.level2_id
-    LEFT JOIN {dbschema}.metadata AS l3
-      ON l3.node_id = ids.level3_id
-    LEFT JOIN {dbschema}.metadata AS l4
-      ON l4.node_id = ids.level4_id
-    LEFT JOIN {dbschema}.metadata AS l5
-      ON l5.node_id = ids.level5_id
-)
-SELECT node_id,
-       title,
-       hr_url,
-       parent_node_id,
-       parent_title,
-       asset_theme_id,
-       asset_subtheme_id,
-       asset_topic_id,
-       asset_subtopic_id,
-       asset_subsubtopic_id,
-       asset_theme,
-       asset_subtheme,
-       asset_topic,
-       asset_subtopic,
-       asset_subsubtopic,
-       sitekey
-FROM biglist
-WHERE index = 1;
-
-ANALYZE {dbschema}.asset_themes;
-
-    """.format(dbschema=dbschema)
+    query = open('ddl/{}'.format(ddl_file), 'r').read()
```

Comment:
https://github.com/bcgov/GDX-Analytics-microservice/blob/1f96d2e846a3e6651f8092712f910aa4d842c6c2/s3_to_redshift/s3_to_redshift.py#L569-L571

Reply:
added a portion of this, but there is no way (in this script) to get to this point if the outfile is not goodfile 
Let me know if this doesnt make sense